tests/backup/backup_helper.go

```diff
@@ -325,49 +326,36 @@ func CreateScheduleBackup(backupName string, clusterName string, bLocation strin
 	}
 	time.Sleep(1 * time.Minute)
 	backupSuccessCheck := func() (interface{}, bool, error) {
-		bkpUid, err = backupDriver.GetBackupUID(ctx, backupName, orgID)
+		schBkpName, err = backupDriver.GetRecentScheduleBackupName(ctx, backupName, orgID)
 		if err != nil {
 			return "", true, err
 		}
-		backupSchInspectRequest := &api.BackupScheduleInspectRequest{
-			Name:  backupName,
-			Uid:   bkpUid,
+		schBkpUid, err = backupDriver.GetRecentScheduleBackupUID(ctx, backupName, orgID)
+		if err != nil {
+			return "", true, err
+		}
+		backupInspectRequest := &api.BackupInspectRequest{
+			Name:  schBkpName,
+			Uid:   schBkpUid,
 			OrgId: orgID,
 		}
-
-		resp, err := backupDriver.InspectBackupSchedule(ctx, backupSchInspectRequest)
+		resp, err := backupDriver.InspectBackup(ctx, backupInspectRequest)
 		if err != nil {
-			return "", true, fmt.Errorf("error in fetching inspect backup schedule response for %v", backupName)
+			return "", true, err
 		}
-		expected := api.BackupScheduleInfo_StatusInfo_Success
-		actual := resp.GetBackupSchedule().GetBackupStatus()["interval"].GetStatus()[0].GetStatus()
+		actual := resp.GetBackup().GetStatus().Status
+		expected := api.BackupInfo_StatusInfo_Success
 		if actual != expected {
 			return "", true, fmt.Errorf("backup status for [%s] expected was [%s] but got [%s]", backupName, expected, actual)
 		}
-		return fmt.Sprintf("actual [%v] is equal to expected [%v] string", actual, expected), false, nil
+		return "", false, nil
 	}
 
 	_, err = task.DoRetryWithTimeout(backupSuccessCheck, 10*time.Minute, 30*time.Second)
 	if err != nil {
 		return "", err
 	}
```

Comment:
Use the helper function `backupSuccessCheck` instead og creating this anonymous function

Reply:
I have added [PA-650](https://portworx.atlassian.net/browse/PA-650) to address this issue couple of days back.